conf/modules.config

```diff
@@ -200,6 +210,57 @@ process {
     }
 }
 
+//
+// Sentieon alignment options
+//
+
+process{
+    withName: '.*ALIGN_SENTIEON:.*' {
+        ext.when = params.aligner.equals("sentieon")
+    }
+
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_BWAMEM' {
+        ext.args = { "-M -K 10000000 -R ${meta.read_group} " }
+        ext.prefix = { "${meta.id}.sorted" }
+        publishDir = [
+            path: { "${params.outdir}/sentieon_align" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
```

Comment:
we shouldn't need that line.
Can we discuss about that in the modules slack channel

Reply:
```suggestion
            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
```